app/ide-desktop/lib/dashboard/src/authentication/src/authentication/providers/auth.tsx

```diff
@@ -25,6 +25,7 @@ const MESSAGES = {
     setUsernameSuccess: 'Your username has been set!',
     signInWithPasswordSuccess: 'Successfully logged in!',
     forgotPasswordSuccess: 'We have sent you an email with further instructions!',
+    changePasswordSuccess: 'Successfully change password!',
```

Comment:
`change` -> `changed`. the `reset` below may be confusing, but it's in past tense - it just happens that the past tense of `reset` is also `reset`